stages/org.osbuild.ostree.deploy

```diff
@@ -84,12 +84,33 @@ SCHEMA_2 = """
 },
 "inputs": {
   "type": "object",
-  "required": ["commits"],
   "additionalProperties": false,
+  "oneOf": [{
+    "required": ["commits"]
+  }, {
+    "required": ["images"]
+  }, {
+      "not": {
+        "anyOf": [{
+            "required": ["commits"]
+          }, {
+            "required": ["images"]
+          }]
+      }
+  }],
   "properties": {
     "commits": {
       "type": "object",
       "additionalProperties": true
+    },
+    "images": {
+      "type": "object",
+      "additionalProperties": true
+    },
```

Comment:
It would be nice to have a description for this object and the previous `commits` object to describe and differentiate their use cases.
As for the commit message for this change, I'd prefer if the example manifest fragments were json objects matching the schema.  `osbuild-mpp` isn't really official API for osbuild and its stages, even if we treat it like that sometimes.  It's more important for contributors to know the json manifest objects rather than the mpp bits.

Reply:
> It would be nice to have a description for this object and the previous `commits` object to describe and differentiate their use cases.

you mean by adding a `"description": "my description"` field?



> As for the commit message for this change, I'd prefer if the example manifest fragments were json objects matching the schema. `osbuild-mpp` isn't really official API for osbuild and its stages, even if we treat it like that sometimes. It's more important for contributors to know the json manifest objects rather than the mpp bits.

would the yaml version suffice?

Reply:
Sure, as long as it's the manifest and not the mpp.

Reply:
I think we've addressed this now.